test/conf/promtool_ignore

```diff
@@ -1,63 +1,52 @@
-templates/alerting-rules/cluster-service.rules.yml
-templates/alerting-rules/coredns.rules.yml
-templates/alerting-rules/credentiald.rules.yml
-templates/alerting-rules/daemonset.management-cluster.rules.yml
-templates/alerting-rules/deployment.management-cluster.rules.yml
-templates/alerting-rules/deployment.workload-cluster.rules.yml
-templates/alerting-rules/dex.rules.yml
-templates/alerting-rules/disk.management-cluster.rules.yml
-templates/alerting-rules/disk.workload-cluster.rules.yml
-templates/alerting-rules/docker.management-cluster.rules.yml
-templates/alerting-rules/docker.workload-cluster.rules.yml
-templates/alerting-rules/etcd.management-cluster.rules.yml
-templates/alerting-rules/etcd.workload-cluster.rules.yml
-templates/alerting-rules/etcdbackup.rules.yml
-templates/alerting-rules/external-dns.rules.yml
-templates/alerting-rules/fairness.rules.yml
-templates/alerting-rules/falco.rules.yml
-templates/alerting-rules/fluentbit.rules.yml
-templates/alerting-rules/helm.rules.yml
-templates/alerting-rules/ingress-controller.rules.yml
-templates/alerting-rules/inhibit.all.rules.yml
-templates/alerting-rules/inhibit.management-cluster.rules.yml
-templates/alerting-rules/job.rules.yml
-templates/alerting-rules/kiam.rules.yml
-templates/alerting-rules/kube-state-metrics.rules.yml
-templates/alerting-rules/kubelet.management-cluster.rules.yml
-templates/alerting-rules/kubelet.workload-cluster.rules.yml
-templates/alerting-rules/kyverno.rules.yml
-templates/alerting-rules/management-cluster.rules.yml
-templates/alerting-rules/microendpoint.rules.yml
-templates/alerting-rules/net-exporter.management-cluster.rules.yml
-templates/alerting-rules/net-exporter.workload-cluster.rules.yml
-templates/alerting-rules/network.all.rules.yml
-templates/alerting-rules/network.management-cluster.rules.yml
-templates/alerting-rules/network.workload-cluster.rules.yml
-templates/alerting-rules/node.management_cluster.rules.yml
-templates/alerting-rules/node.workload_cluster.rules.yml
-templates/alerting-rules/operatorkit.rules.yml
-templates/alerting-rules/prometheus-meta-operator.rules.yml
-templates/alerting-rules/prometheus-operator.rules.yml
-templates/alerting-rules/release.rules.yml
-templates/alerting-rules/secret.rules.yml
-templates/alerting-rules/service-level.rules.yml
-templates/alerting-rules/systemd.management-cluster.rules.yml
-templates/alerting-rules/systemd.workload-cluster.rules.yml
-templates/alerting-rules/timesync.rules.yml
-templates/alerting-rules/up.management-cluster.rules.yml
-templates/alerting-rules/vault.rules.yml
-templates/recording-rules/grafana-cloud.rules.yml
-templates/recording-rules/gs-managed-app-deployment-status.rules.yml
-templates/recording-rules/kubernetes-mixins.rules.yml
-templates/recording-rules/service-level.rules.yml
-templates/recording-rules/mimir-mixins.rules.yml
-templates/recording-rules/loki-mixins.rules.yml
+alerting-rules/areas/kaas/bigmac/dex.rules.yml
```

Comment:
I'm confused. If we have areas/kaas/ the rest should be team/bigmac to be area/kaas/team/bigmac or just kaas/bigmac but having area and not team is weird.

@giantswarm/team-atlas I also think we should have area/team/recording|alerting and not alerting/area/team. WDYT?

Reply:
I removed `team` in the path because it was obvious for me.
Area gives more context for `kaas` and `platform` values IMHO.

For moving alerting and recording under each team, I think about it but I wanted the minimal number of changes in that PR and currently our recording rules are multi-teams.

Reply:
We need to align imo

Reply:
So yes it can be minimal changes, but I would rather we have alerting-rules at the end of the tree now and we move recording rules later

Reply:
The path in the helm chart is:
`helm/prometheus-rules/templates/alerting-rules/areas/[kaas|platform|shared]/[team]`
`helm/prometheus-rules/templates/recording-rules`

Having `alerting-rules` under team will result to add it multiple times.

Reply:
Well we will then have `helm/prometheus-rules/templates/recording-rules/areas/[kaas|platform|shared]/[team]` later so 🤔

Reply:
Yes it's my idea.
But for that PR as we don't have recording rules per team, I didn't change the behavior for them.

Reply:
I agree with Quentin, I think it would be better to have a path like this `areas/<area_name>/<team_name>/<alerting-rules|recording-rules>` so that for a given team folder, the team has access directly to both its alerting and recording rules.

Reply:
So we we either have alerting-rules duplicated under each team, or each team duplicated under alerting-rules and recording rules. I personnally prefer the first one because all rules are in the same place. We could even add a folder for slos so we could have:

helm/prometheus-rules/templates/areas/[kaas|platform|shared]/[team]/{alerting-rules|recording-rules|slos}

Reply:
Also, I think shared should only be temporary but we can slowly get rid of it

Reply:
I don't like the repetition of alerting-rules and recording-rules in all teams.
Maybe we can put all rules in team folder and prefix the name of the recording-rules to identify them ?

Reply:
But then you liké to have team name duplicated? I'm not sure I follow the Logic :)

Reply:
Yes you're right.
My idea was more when recording-rules will be specific per team.
At this time, we will be able to move them in the same directory and get rid of alerting-rules/recording-rules folders.

Reply:
Let's discuss in planning :)